md-dao/src/main/java/com/wipro/ats/bdre/md/dao/GetETLInfoDAO.java

```diff
@@ -55,13 +55,13 @@
                     batchList.add(batchSizeList);
                 }
             }
-            if (batchList.size() != 0) {
+            if (!batchList.isEmpty()) {
                 Criteria FileCriteria = session.createCriteria(File.class).add(Restrictions.in("batch", batchList));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00117)